data.sql

```diff
@@ -46,4 +46,37 @@ update animals set owner_id = '1' where name = 'Agumon'
 update animals set owner_id = '2' where name = 'Gabumon' or name = 'Pikachu'
 update animals set owner_id = '3' where name = 'Devimon' or name = 'Plantmon'
 update animals set owner_id = '4' where name = 'Charmander' or name = 'Squirtle' or name = 'Blossom'
-update animals set owner_id = '5' where name = 'Angemon' or name = 'Boarmon'
\ No newline at end of file
+update animals set owner_id = '5' where name = 'Angemon' or name = 'Boarmon'
+--insert data into vets table
+INSERT INTO vets(name, age, date_of_graduation)VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+--insert data into specialization table
+INSERT INTO specialization(species_id, vets_id)VALUES
+(1, 1),
+(2, 3),
+(1, 3),
+(2, 4);
+--insert data into visits table
```

Comment:
* [OPTIONAL] This insertion could have been done dynamically where you do not know the `vet_id` and `specie_id`.

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```